twitter-borrowbot/twitterbotErrors.py

```diff
@@ -0,0 +1,123 @@
+class Error(Exception):
+    pass
+
+class TweepyAuthenticationError(Error):
+    def __init__(self, error=None):
+        self.error = error
+
+    def __str__(self):
+        return "{0}: Failed to Authenticate with Twitter through Tweepy >> {1}".format(type(self).__name__, self.error)
+
+class FileIOError(Error):
+    def __init__(self, file=None, write=None, error="Error not provided"):
+        self.file = file
+        self.write = write
+        self.error = error
+
+    def __str__(self):
+        if self.write:
+            return "{0}: Failed to write '{1}' to file '{2}' >> {3}".format(type(self).__name__, self.write, self.file, self.error) 
+        return "{0}: Failed to read from file '{1}' >> {2}".format(type(self).__name__, self.file, self.error)
```

Comment:
The numbers inside {} have not been needed since Python 2.6 and in general are dropped.  The use of Python f-strings would be shorter and faster.
```suggestion
            return f"{type(self).__name__,}: Failed to write '{self.write}' to file '{self.file}' >> {self.error}"
        return f"{type(self).__name__}: Failed to read from file '{self.file}' >> {self.error}"
```

Reply:
Yes that was my thought too. I was just conforming to the original writers style. I will swap over.